app/static/js/financialAidOverload.js

```diff
@@ -58,3 +58,21 @@ function openApproveDenyModal(status){
     $("#finOverloadModal").modal("show");
   }
 }
+
+function overloadNoteInsert(textareaID, buttonID) {
```

Comment:
A good variable name, ideally verbs come first. I would recommend the variable be switched to insertOverloadNote.